terraform/nifikop.tf

```diff
@@ -0,0 +1,50 @@
+# Create NiFikop namespace if asked
+resource "kubernetes_namespace" "nifikop" {
+  count = var.nifikop_create_namespace ? 1 : 0
+  metadata {
+    annotations = {
+      name = var.nifikop_namespace
+    }
+    name = var.nifikop_namespace
+  }
+}
+
+# Deploy CRDs
+resource "k8s_manifest" "nifikop_crds" {
```

Comment:
These will be automatically deployed along with the nifikop helm chart, so I don't believe it's necessary to deploy them alongside the chart. Unless this is just to be able to delete them from the Kubernetes cluster.

Reply:
It is necessary to update the CRDs, as helm is not able to update the existing CRDs, we have to deploy them explicitly in terraform, this way if we deploy a new version of CRDs they will be deployed.